tests/test_setup_utils.py

```
@@ -194,40 +194,75 @@ def test_wait_for_service_start_full_duration(self, mock_sleep, mock_is_active):
         mock_read_service.return_value = "existing service content"
         mock_needs_update.return_value = (True, "Executable path changed")
+        mock_create_service_file.return_value = True
+        mock_reload_daemon.return_value = True
+        mock_loginctl_available.return_value = False
+        mock_is_enabled.return_value = True
+        mock_is_active.return_value = False
         mock_input.return_value = "n"
 
         result = install_service()
 
         self.assertTrue(result)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)